### ISSUE TITLE
Favourites API endpoint is not sorted by time

### ISSUE THREAD

**Issue by Jonty** [emfcamp/Website#1732]
We had people complaining about this and it would make more sense in terms of usability. Presumably this is also true of the non-schedule lineup favourites view.

**Comment by lukegb** [comment]
We probably want this to have different behaviour when the schedule is unveiled vs. beforehand to stop a repeat of people trying to plan things before the schedule is finalized.

**Comment by Jonty** [comment]
Agreed! That might be resolved by all the schedule timing rework, but we should keep it in mind.
